xapi-consts.opam.template

```diff
@@ -11,7 +11,7 @@ build: [
 depends: [
   "ocaml"
   "dune" {build & >= "1.4"}
-  "dune-build-info"
+  ""
```

Comment:
```suggestion
```

Reply:
Aha, this was the actual reason for the `dune-build-info` build failure.
I was overly eager to remove it with `git sed`, but it does need to stay in this `opam` file, otherwise it is completely absent from the build environment.

It all builds fine locally because I already had it installed, and koji uses a previous xs-opam so it'd also have it installed.